dns/zone-configs/k8s.io.yaml

```diff
@@ -248,3 +248,7 @@ kind.sigs:
 cluster-api.sigs:
   type: CNAME
   value: kubernetes-sigs-cluster-api.netlify.com.
+# https://github.com/kubernetes/cloud-provider-vsphere docs (@andrewsykim, @frapposelli)
+vsphere.sigs:
```

Comment:
Oh btw this will also need to be `cloud-provider-vsphere.sigs` to stay consistent with the naming... :grimacing:

Reply:
`cloud-provider-vsphere.sigs` is harder to remember than `vsphere.sigs`, but I see the importance of consistency as well :P. Will update shortly

Reply:
okay done!